services/userService.js

```diff
@@ -0,0 +1,69 @@
+const jwt = require('jsonwebtoken');
+const axios = require('axios');
+const userDao = require('../models/userDao');
+
+const getKakaoUserInfo = async (kakaoAccessToken) => {
+  let kakaoUserInfo;
+
+  await axios({
+    method: 'POST',
+    url: 'https://kapi.kakao.com/v2/user/me',
+    headers: {
+      'Content-type': 'application/x-www-form-urlencoded;charset=utf-8',
+      'Authorization': `Bearer ${kakaoAccessToken}`
+    }
+    }).then((result) => {
+      kakaoUserInfo = result.data;
+    })
+  
+  return kakaoUserInfo
+};
+
+const storeKakaoUserInfo = async(kakaoUserInfo) => {
+  const kakaoId = kakaoUserInfo.id;
+  const [ userId ] = await userDao.getUserIdByKakaoId(kakaoId);
+
+  if (!userId) {
+    await userDao.storeKakaoUserInfo(kakaoUserInfo)
+  }
+};
+
+const generateToken = async(kakaoUserInfo) => {
+  const [ userId ] = await userDao.getUserIdByKakaoId(kakaoUserInfo.id);
+
+  if (!userId) {
+    const err = new Error('SPECIFIED_USER_DOES_NOT_EXIST');
+    err.statusCode = 404;
+    throw err;
+  }
+
+  return jwt.sign({ userId: userId.userId }, process.env.JWT_SECRET, { expiresIn: process.env.JWT_EXPIRES_IN });
```

Comment:
jwt 해싱을 위한 알고리즘 값 설정을 해주세요!
```suggestion
  return jwt.sign({ userId: userId.userId }, process.env.JWT_SECRET,
    {
      algorithm: process.env.ALGORITHM,
      expiresIn: process.env.JWT_EXPIRES_IN
    });
```

Reply:
알고리즘 값 설정을 위해 코드를 추가하겠습니다.
.env.sample 파일에도 알고리즘 값을 기재하여 push 하겠습니다!